ajax/libs/botui/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "botui",
+  "description": "A JS library to build the UI for your bot",
+  "filename": "botui.min.js",
+  "keywords": [
+    "bot",
+    "chat",
+    "conversational",
+    "interface",
+    "js",
+    "ui"
+  ],
+  "author": {
+    "name": "Moin Uddin",
+    "url": "https://moin.im"
+  },
+  "homepage": "https://botui.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/botui/botui"
+  },
+  "autoupdate": {
```

Comment:
Thanks @TechieBoy 😄 
Let's use npm auto-update instead. Because there are more versions on npm.
It would be like
```javascript
  "npmName": "botui",
  "npmFileMap": [
    {
      "basePath": "build",
      "files": [
        "**/*"
      ]
    }
  ]
```
`files` is a pattern matcher allowing selection of multiple files, you can see [node-glob](https://github.com/isaacs/node-glob). That's why I use `**/*` here.
For more details about auto-update, you can see https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md